datasources.go

```diff
@@ -36,6 +42,16 @@ func (api API) CreateDataSource(name string) (*DataSource, error) {
 	return ds, err
 }
 
+// CreateDataSource2 creates an API Data Source in ChartMogul.
+// * Allows other parameters than just the name.
+//
+// See https://dev.chartmogul.com/v1.0/reference#data-sources
+func (api API) CreateDataSource2(dataSource *DataSource) (*DataSource, error) {
```

Comment:
@pkopac IIUC, do `CreateDataSourceWithSystem` and `ListDataSourcesWithFilters` make sense? Long, but at least a name that states what it does.